requirements/optional.txt

```diff
@@ -2,4 +2,4 @@ matplotlib
 dipy>=1.0.0
 Cython
 jupyter-rfb>=0.4.4
-PySide6>=6.8.2.1
\ No newline at end of file
+PySide6>=6.8.2.1
```

Comment:
do not forget the `pyproject.toml`

Reply:
It has been added in the window section in the `pyproject.toml`